AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Dependency/AuthorizationProviderAdapter+SignedInSession.swift

```diff
@@ -112,9 +112,7 @@ extension AuthorizationProviderAdapter {
                 if let urlError = error as NSError?, urlError.domain == NSURLErrorDomain {
                     self.fetchSignedInSessionWithOfflineError(completionHandler)
 
-                } else if let cognitoIdentityPoolError = error as NSError?,
-                    cognitoIdentityPoolError.domain == AWSCognitoIdentityErrorDomain,
-                    cognitoIdentityPoolError.code == AWSCognitoIdentityErrorType.notAuthorized.rawValue {
+                } else if !self.identityPoolConfigured(error: error!) {
```

Comment:
More context would be handy here. Why is it safe to ignore what appears just be reading code to be a misconfigured identity pool? Changing the method name to self-document the intent of the check would be helpful, but honestly I can't tell what the method below is actually trying to do.

Reply:
Changed the method name